components/rhel/rhel-ose/Vagrantfile

```diff
@@ -75,7 +75,12 @@ Vagrant.configure(2) do |config|
 
   # explicitly enable and start OpenShift
   config.vm.provision "shell", run: "always", inline: <<-SHELL
-    PROXY=#{PROXY} PROXY_USER=#{PROXY_USER} PROXY_PASSWORD=#{PROXY_PASSWORD} IMAGE_TAG=#{IMAGE_TAG} /usr/bin/sccli openshift
+    if [ -z #{IMAGE_TAG} ]
+    then
+      PROXY=#{PROXY} PROXY_USER=#{PROXY_USER} PROXY_PASSWORD=#{PROXY_PASSWORD} /usr/bin/sccli openshift
+    else
+      PROXY=#{PROXY} PROXY_USER=#{PROXY_USER} PROXY_PASSWORD=#{PROXY_PASSWORD} IMAGE_TAG=#{IMAGE_TAG} /usr/bin/sccli openshift
+    fi
```

Comment:
@LalatenduMohanty since now sccli is capable of handling null variable I think we can go back older solution and remove if-else condition.